src/DotNetLightning.Core/Channel/Commitments.fs

```diff
@@ -183,3 +183,46 @@ type Commitments = {
             match remoteSigned, localSigned with
             | Some _, Some htlcIn -> htlcIn.Add |> Some
             | _ -> None
+
+        member this.SpendableBalance(): LNMoney =
+            let remoteCommit =
+                match this.RemoteNextCommitInfo with
+                | RemoteNextCommitInfo.Waiting info -> info.NextRemoteCommit
+                | RemoteNextCommitInfo.Revoked _info -> this.RemoteCommit
+            let reducedRes =
+                remoteCommit.Spec.Reduce(
+                    this.RemoteChanges.ACKed,
+                    this.LocalChanges.Proposed
+                )
+            let reduced =
+                match reducedRes with
+                | Error err ->
+                    failwithf
+                        "reducing commit failed even though we have not proposed any changes\
+                        error: %A"
+                        err
+                | Ok reduced -> reduced
+            let fees =
+                if this.LocalParams.IsFunder then
+                    Transactions.commitTxFee this.RemoteParams.DustLimitSatoshis reduced
+                    |> LNMoney.FromMoney
+                else
+                    LNMoney.Zero
+            let channelReserve =
+                this.RemoteParams.ChannelReserveSatoshis
+                |> LNMoney.FromMoney
+            let totalBalance = reduced.ToRemote
+            let untrimmedSpendableBalance = totalBalance - channelReserve - fees
+            let htlcSuccessFee =
+                reduced.FeeRatePerKw.ToFee(Transactions.Constants.HTLC_SUCCESS_WEIGHT)
```

Comment:
why do we use any HTLC-related stuff here if we don't use HTLCs?

Reply:
Because I had to think hard about how exactly this function needs to be implemented and I didn't want to have to do that again, or to have the code silently break, when we eventually add support for HTLCs. So I just made it HTLC-compatible to begin with.

Reply:
that's great but I'd prefer if you move this code into another PR, to pick it up later

Reply:
I've removed the HTLC fees from the calculation and created a seperate PR which adds them back here: #3